test/acceptance/tests/basic/terraform/pass-existing-iam-roles/main.tf

```diff
@@ -1,22 +1,70 @@
-// We test this with a Terraform plan only.
+// We test this with a 'terraform apply'.
+// It creates roles and the task definition.
 
 provider "aws" {
   region = "us-west-2"
 }
 
-module "test_client" {
-  source = "../../../../../../modules/mesh-task"
-  family = "family"
+output "suffix" {
+  value = local.suffix
+}
+
+output "create_roles_family" {
+  value = local.create_roles_family
+}
+
+output "pass_roles_family" {
+  value = local.pass_roles_family
+}
+
+output "create_roles_task_definition_arn" {
+  value = module.test_client_create_new_roles.task_definition_arn
+}
+
+output "pass_roles_task_definition_arn" {
+  value = module.test_client_pass_existing_roles.task_definition_arn
+}
+
+
+resource "random_string" "suffix" {
+  length  = 8
+  special = false
+}
+
+locals {
+  suffix              = lower(random_string.suffix.result)
+  create_roles_family = "consul-ecs-test-create-roles-${local.suffix}"
+  pass_roles_family   = "consul-ecs-test-pass-existing-roles-${local.suffix}"
   container_definitions = [{
-    name = "basic"
+    name  = "basic"
+    image = "fake"
   }]
-  retry_join    = ["test"]
-  outbound_only = true
+}
+
+module "test_client_create_new_roles" {
+  source                = "../../../../../../modules/mesh-task"
+  family                = local.create_roles_family
+  log_configuration     = null
```

Comment:
minor: default for `log_configuration` is `{}` instead of `null`.. we probably don't need to pass this here and can just get rid of this line.

Reply:
Um. When I remove `log_configuration` I get the following error. 

For now, I'm going to leave this as-is, but this is something we should fix somewhere so I added an [Asana task](https://app.asana.com/0/1200191208677272/1202513121164993) for it.

```
Error: failed creating ECS Task Definition (consul-ecs-test-create-roles-oepmkxig): InvalidParameter: 3 validation error(s) found.
- missing required field, RegisterTaskDefinitionInput.ContainerDefinitions[1].LogConfiguration.LogDriver.
- missing required field, RegisterTaskDefinitionInput.ContainerDefinitions[2].LogConfiguration.LogDriver.
- missing required field, RegisterTaskDefinitionInput.ContainerDefinitions[3].LogConfiguration.LogDriver.
```